docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14.5
+
+RUN apt-get install -y nodejs 
```

Comment:
You still don't need this if you're using the node as the base image

Reply:
Yes sorry about that, I forgot removing it. I have corrected the Dockerfile now. Its running for development.